esp/src/lws.config.js

```diff
@@ -1,4 +1,7 @@
-const debugServerIP = "192.168.99.103";
```

Comment:
Not sure these should be in a checked in file?

Reply:
I concur (I am not sure either) - but it is really handy having them...

Reply:
(I can compromise by only keeping the 192.* ones?)